time-sync-example/time-sync-configure-ops-agent-monitor.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+curl -sSO https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh
+bash add-google-cloud-ops-agent-repo.sh --also-install
```

Comment:
We are suggesting to run a script that we download from the network. Is there another installation option that doesn't involve that users blindly run a script about which they don't know anything?

If there's no other option, we should at least give users access to the script source code.